src/playground/index.jsx

```diff
@@ -28,6 +28,13 @@ document.body.appendChild(appTarget);
 GUI.setAppElement(appTarget);
 const WrappedGui = HashParserHOC(AppStateHOC(GUI));
 
+// simple example of how you might manage project title externally.
+// Changing project title within GUI interface will update it here.
+let projectTitle = 'Untitled-1';
+const onUpdateProjectTitle = title => {
+    projectTitle = title;
```

Comment:
Does this really cause the GUI to re-render? I would expect this to happen silently, so you would need to have a wrapper component that held this in state (like in the player example).

Reply:
NO it doesn't! Fixed.